lib/utils/report.mjs

```diff
@@ -115,6 +125,45 @@ export default function report(problem) {
 	result.warn(warningMessage, warningProperties);
 }
 
+/**
+ * @param {Problem} problem
+ */
+function problemRangeDeprecations(problem) {
+	// TODO: `config.quietDeprecationWarnings` is not available in `problem`
+
+	if (!problem.node) {
+		emitDeprecationWarning(
+			`Omitting \`node\` in \`report\` is deprecated ("${problem.ruleName}").`,
```

Comment:
[suggestion] How about clarifying `node` and `report` like this? It may be a bit longer, though:

```suggestion
			`Omitting the \`node\` argument in the \`report()\` function is deprecated ("${problem.ruleName}").`,
```

For example:

https://github.com/stylelint/stylelint/blob/003bbc416c48e5bf774f32b833aaa0d2259950bc/lib/utils/report.mjs#L66

Reply:
Good suggestion!
I've made some changes, please let me know if these are sufficiently clear now.

Reply:
Thank you. For consistency, could you change `report`s in the `emitDeprecationWarnings` third arguments, too?